nixos/doc/manual/release-notes/rl-2103.xml

```diff
@@ -264,6 +264,19 @@
      <literal>unbound-control</literal> without passing a custom configuration location.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     NixOS now defaults to the unified cgroup hierarchy (cgroupsv2), which has been made a default
+     in other distributions, like Fedora 32.
+     See the <link xlink:href="https://www.redhat.com/sysadmin/fedora-31-control-group-v2">Fedora Article for 32</link>
+     for details on why this is desirable, and how it impacts containers.
```

Comment:
I'm not sure we need to appeal to other distros as an "authority".  How about something like:

```suggestion
     NixOS now defaults to the unified cgroup hierarchy (cgroupsv2).
     See the <link xlink:href="https://www.redhat.com/sysadmin/fedora-31-control-group-v2">Fedora Article for 32</link>
     for details on why this is desirable, and how it impacts containers.
```

Reply:
Updated, thanks!